pages/verify-identity.tsx

```diff
@@ -48,7 +47,7 @@ export default function IdentityVerificationForm() {
 
   const [userId, setUserId] = useState('');
   const [phoneNumberSuffix, setPhoneNumberSuffix] = useState('');
-  const [dateOfBirth, setDateOfBirth] = useState(formatDate(new Date()));
+  const [dateOfBirth, setDateOfBirth] = useState(new Date().toISOString().substring(0, 10));
```

Comment:
See above comment. The `formatDate` function displays yyyy-mm-dd instead of pre-filling with the current date which was the original behavior